react/features/transcribing/actions.ts

```diff
@@ -57,59 +51,3 @@ export function potentialTranscriberJoined(participantId: string) {
         transcriberJID: participantId
     };
 }
-
-/**
- * Signals that the pending transcribing notification should be shown on the
- * screen.
- *
- * @returns {showNotification}
- */
-export function showPendingTranscribingNotification() {
-    return showNotification({
-        descriptionKey: 'transcribing.pending',
```

Comment:
do we care about cleaning some of these translations if they're not used anymore?

Reply:
Yeah forgot about that, thanks for reminding me!

Reply:
Looks like I had done it already 😅  https://github.com/jitsi/jitsi-meet/pull/14234/files#diff-acc6f5186c27e0f1c972135d39d2567f7deea1a3993c13eb8df685a955caa869L1362